client/src/components/LoginComponent/login.js

```diff
@@ -1,24 +1,51 @@
 import React from "react";
 import "./login.css";
+import axios from "axios";
+class Login extends React.Component {
+  handleSubmit(event) {
+    event.preventDefault();
+    const { username, password } = event.target.elements;
+    const postData = {
+      username: username.value,
+      password: password.value
+    };
+    axios
+      .post("/login", postData)
+      .then(res => console.log(res))
```

Comment:
Is this going to be changed to something else later? logging will not be useful in the future

Reply:
sure it will .. just used  it for checking my code ! :)